src/auth/permissions/__tests__/index.test.ts

```diff
@@ -0,0 +1,187 @@
+import { permissionsInitialiser } from '../index';
+const { getConfig } = require('../../../../config');
+const { migrate } = require('../../../../database');
+import * as knexInit from 'knex';
+
+describe('Permisions Module', () => {
+  const config = getConfig(process.env.NODE_ENV);
+  const knex = knexInit(config.knex);
+  const permissionsInterface = permissionsInitialiser(knex);
+  beforeAll(async() => {
+    await migrate.teardown({ client: knex });
+    await knex.migrate.latest();
+    await knex.seed.run();
```

Comment:
Minor point, but I think we can remove this, since it'll be run in `beforeEach` anyway